src/plots/cartesian/axis_defaults.js

```diff
@@ -69,15 +67,22 @@ module.exports = function handleAxisDefaults(containerIn, containerOut, coerce,
     // try to get default title from splom trace, fallback to graph-wide value
     var dfltTitle = splomStash.label || layoutOut._dfltTitle[letter];
 
+
+    handleTickLabelDefaults(containerIn, containerOut, coerce, axType, options);
```

Comment:
Unfortunately, there's a bunch of attributes that get coerced in `handleTickLabelDefaults` that will have no effect when `ax.visible:false`.

I suggest moving back  `handleTickLabelDefaults` back below the `if(!visible) ...`  statement and moving the `coerce()` calls for `tickformat`, `tickprefix` and `ticksuffix` here above the `if(!visible) ...`

Reply:
Good call. Revised in https://github.com/plotly/plotly.js/pull/3817/commits/e968da68abbc70c67a0b08f646d266a2e1754dd2.

Reply:
Nicely done :ok_hand: